plugins/modules/foreman_operating_system.py

```diff
@@ -192,81 +184,80 @@ def main():
             ptables=dict(type='list'),
             provisioning_templates=dict(type='list'),
             password_hash=dict(choices=['MD5', 'SHA256', 'SHA512']),
-            parameters=dict(type='dict'),
+            parameters=dict(type='list', elements='dict', options=dict(
```

Comment:
this is a backwards incompatible change. I think I'm okay with it as we don't promise a stable API (yet), but I wanted to point it out :)

Reply:
Right. It is kind of tied to the api change of parameters in foreman 1.22.

Reply:
But the module remains compatible with 1.21 and older because our magic will just throw away that type param if its not in the apidoc, right?

Reply:
That's what we all hope for. ;)